### PR TITLE
Feature/618 fixes of data fetch 10 2021

### PR DESCRIPTION
**Pull request by sinejespersen**
https://jira.itkdev.dk/browse/AR-688

### CODE REVIEW

File: src/components/playlist-drag-and-drop/playlist-drag-and-drop.jsx [hunk 1]
```diff
@@ -49,6 +49,15 @@ function PlaylistDragAndDrop({ handleChange, name, data }) {
     setDataStructureToDisplay();
   }
 
+  /**
+   * Fetches data for the multi component // @TODO:
+   *
+   * @param {string} filter - the filter.
+   */
+  function onFilter(filter) {
+    console.log(filter);
```
Comment (tuj) [inline src/components/playlist-drag-and-drop/playlist-drag-and-drop.jsx:58]: Why console.log ?

File: src/components/groups/group-create.jsx [hunk 2]
```diff
@@ -17,8 +17,8 @@ function GroupCreate() {
   const [formStateObject, setFormStateObject] = useState({
     title: "",
     description: "",
-    createdBy: "todo",
-    modifiedBy: "todo",
+    createdBy: "@TODO:",
```
Comment (tuj) [inline src/components/groups/group-create.jsx:20]: Will it cause an error in the API to not pass a valid date?

File: src/components/screen/screen-form.jsx [hunk 3]
```diff
@@ -44,31 +41,24 @@ function ScreenForm({
...
+   *
+   * @param {string} filter - the filter.
    */
-  useEffect(() => {
-    if (data && !Array.isArray(screen.inScreenGroups)) {
-      setSelectedGroups(data["hydra:member"]);
-    }
-  }, [data]);
+  function onFilter(filter) {
+    console.log(filter);
```
Comment (tuj) [inline src/components/screen/screen-form.jsx:60]: Console.log...

File: src/components/slide/slide-form.jsx [hunk 4]
```diff
@@ -53,6 +61,37 @@ function SlideForm({
...
+    }
+  }, [template]);
+
+  /**
+   * Fetches data for the multi component // @TODO:
+   *
+   * @param {string} filter - the filter.
+   */
+  function onFilter(filter) {
+    console.log(filter);
```
Comment (tuj) [inline src/components/slide/slide-form.jsx:79]: Console.log...

File: src/components/groups/group-create.jsx (hunk 2 above)
Comment (sinejespersen) [reply src/components/groups/group-create.jsx:20]: At one point it did, I am not sure if it does now.

**Pull request merged**